cli/src/pcluster/utils.py

```diff
@@ -36,18 +36,48 @@
 
 LOGGER = logging.getLogger(__name__)
 
+DEFAULT_PARTITION = "aws"
+PARTITION_MAP = {
+    "cn": "aws-cn",
```

Comment:
why not `cn-?`

Reply:
No specific reason. I agree to use `cn-` instead of `cn`.